crates/vm-superio/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+# v0.6.0
+
+## Added
+
+- Added `SerialState`, and support for saving and restoring the state of the
+  `Serial` device ([#73](https://github.com/rust-vmm/vm-superio/pull/73)).
+- Added the `Debug` derive to the exported structures
+  ([#75](https://github.com/rust-vmm/vm-superio/pull/75)).
+
+## Fixed
+
+- Fixed a corner case of the serial's `enqueue_raw_bytes` method
```

Comment:
I think it would be better to have short description here instead, something like Fixed `enqueue_raw_bytes` for the corner case in which 0 bytes were sent.

Reply:
done